### ISSUE TITLE
rubocop linting itself

### ISSUE THREAD

**Issue by zkoppert** [github/super-linter#382]
A bit of an update for this: it seems to be working better with no `RUBY_CONFIG_FILE`, but a `.ruby-lint.yml -> .rubocop.yml` symlink as originally reported so I wonder if this is a different issue? It's still not working correctly though, it seems to be linting rubocop itself in that case:
```
ERROR! Found errors in [rubocop] linter!
5
ERROR:[/usr/lib/ruby/gems/2.7.0/gems/rubocop-0.74.0/lib/rubocop/config_loader_resolver.rb:76: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
6
/usr/lib/ruby/gems/2.7.0/gems/rubocop-0.74.0/lib/rubocop/config_loader_resolver.rb:84: warning: The called method `merge' is defined here
7
Inspecting 1 file
```
So possibly this is yet another issue.

_Originally posted by @DawidJanczak in https://github.com/github/super-linter/issues/282#issuecomment-654047050_

**Comment by GaboFDC** [comment]
Hey @DawidJanczak can you clarify about this? and test if there is still a problem in the latest version?

**Comment by DawidJanczak** [comment]
Hello @GaboFDC I just tested on the newest published version (3.3.0) and it seems to be all working fine now! Thanks, this can be closed :)
